api-extractor/src/IDocItem.ts

```diff
@@ -141,7 +141,8 @@ export interface IDocMethod extends IDocBase {
   /**
    * a mapping of parameter name to IDocParam
    */
-  parameters: Map<string, IDocParam>;
+
+  parameters: { [name: string]: IDocParam};
```

Comment:
I would use `Object.keys()` personally. Eliminates the need for the hasOwnProperty.

However, it is slightly less performant for massive lists.

https://jsperf.com/sumof-forin-vs-object-keys